### DISCUSSION TITLE
Where could I find the metabolism fraction of total hepatic clearance?

### DISCUSSION THREAD

**Discussion by Harris728** [Open-Systems-Pharmacology/Forum#1224] [Questions & Problems]
Hello all, 

If I defined a "total hepatic clearance" for a compound, where could I find the fraction of the compound eliminated by "total hepatic clearance"?

Thanks.

**Comment by Prajwala1921** [comment]
Dear @Harris728 
Can you please elaborate the question?  Do you mean, you gave hepatic plasma clearance along with other clearance mechanism or hepatic clearance alone in your model?? Usually if specific enzyme data  is not available. Then we can use mass balance data to quantitate the hepatic, renal and bile excretion from total clearance reported or calculated NCA (non compartmental analysis )clearance.  In worst case, you can use the NCA calculated clearance as total plasma clearance if mass balance data is not available.

**Reply by Harris728** [reply]
Hello @Prajwala1921 
Thank you for your response. We have a compound metabolized by CYP2C19 and amidase and have developed a PBPK model for it. The metabolism by CYP2C19 was already defined in PK-sim, and we applied "total hepatic clearance" as a replacement of amidase. Now, we want to know the fraction of metabolism by amidase (total hepatic clearance) in the compound elimination. Where could I find it in the PBPK model?

**Comment by prvmalik** [comment]
Try using the observer under "Define Settings and Run" > "Organism" > "Total Fraction of Dose - Metabolite" or a similar name to this effect.

I would have to check, but if this doesn't work then I would avoid using the 'total hepatic clearance' parameterization and instead insert a dummy enzyme in the liver intracellular space. An observer would certainly work in that case.

**Reply by Harris728** [reply]
Hello @prvmalik 
I have tried the former method, but it doesn't work. There is only CYP2C19 relevant metabolism fraction. As for the second method, how could I insert a dummy enzyme. I can't find the corresponding enzyme in the enzyme pool in PK-sim, and I don't know the abundance of this enzyme.

**Reply by prvmalik** [reply]
You would insert a 'default' enzyme and name it with concentration 1 umol/L in the liver intracellular space. Then optimize the first order specific clearance against observed data or manually tune the specific clearance to achieve the desired plasma clearance. I recommend the "in vitro specific clearance - first order" setup for your enzyme.

Alternatively, if there is no biliary excretion then the fraction eliminated by your non-specific hepatic plasma clearance can be calculated as 1 - fm,CYP2C19 - fe,urine - fe,feces, which are all observers that are available.

**Reply by Harris728** [reply]
Thank you so much for your great answer. It really helps me a lot.
